plugins/auth-backend-module-cloudflare-access-provider/src/helpers.ts

```diff
@@ -40,6 +40,10 @@ export class AuthHelper {
     options?: { cache?: CacheService },
   ): AuthHelper {
     const teamName = config.getString('teamName');
+    const jwtHeaderName = config.getOptionalString('jwtHeaderName');
+    const authorizationCookieName = config.getOptionalString(
+      'authorizationCookieName',
+    );
```

Comment:
Let's do the defaulting here instead to avoid future mistakes, for example `config.getOptionalString('jwtHeaderName') ?? CF_JWT_HEADER`

Reply:
Ok done.